### PR TITLE
Add logging for aot autograd and unified debug flag

### PR DESCRIPTION
**Pull request by mlazos**
- Adds `log_level` to aot's config
- Outputs log to `<graph_name>_<log_level>.log` in aot_torchinductor subfolder of the debug directory
- Modifies the Inductor debug context to use the graph name when naming the folder instead of the os pid
- Adds `TORCH_COMPILE_DEBUG` flag to enable it, (as well as separate dynamo and inductor logs)

cc @gujinghui @PenghuiCheng @XiaobingSuper @jianyuh @VitalyFedyunin @jgong5 @mingfeima @sanchitintel @ashokei @jingxu10 @min-jean-cho @yanbing-j @Guobing-Chen @Xia-Weiwen @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @chunyuan-w @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: functorch/_src/config.py [hunk 1]
```diff
@@ -8,20 +8,25 @@
 Global flags for aot autograd
 """
 import os
+import logging
 
 use_functionalize = True
 
 # TODO Benchmark
 use_fake_tensor = False
```
Comment (mlazos) [inline functorch/_src/config.py:17]: Need to fix this config

**Review by Chillee — APPROVED** [review]
Mostly LGTM. Some notes:

1. I might have missed something since I think formatting changes got bundled up.
2. Do we still need *both* TORCHINDUCTOR_TRACE and TORCH_COMPILE_DEBUG?

**Comment by mlazos** [comment]
> Mostly LGTM. Some notes:
> 
> 1. I might have missed something since I think formatting changes got bundled up.
> 2. Do we still need _both_ TORCHINDUCTOR_TRACE and TORCH_COMPILE_DEBUG?

Thanks for taking a look, so we don't, but I left TORCHINDUCTOR_TRACE to not trip everyone up, I can remove it and I'll announce it before merging

**Comment by mlazos** [comment]
I'm going to make the graphs runnable in a separate PR, I need to untangle the minifier infra a bit and it's turning into a large change

**Pull request closed without merging**